pkg/tbtc/inactivity.go

```diff
@@ -0,0 +1,263 @@
+package tbtc
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"math/big"
+	"sync"
+
+	"github.com/ipfs/go-log/v2"
+	"go.uber.org/zap"
+	"golang.org/x/sync/semaphore"
+
+	"github.com/keep-network/keep-core/pkg/bitcoin"
+	"github.com/keep-network/keep-core/pkg/chain"
+	"github.com/keep-network/keep-core/pkg/generator"
+	"github.com/keep-network/keep-core/pkg/net"
+	"github.com/keep-network/keep-core/pkg/protocol/group"
+	"github.com/keep-network/keep-core/pkg/protocol/inactivity"
+)
+
+const (
+	// inactivityClaimSubmissionDelayStepBlocks determines the delay step in blocks
+	// that is used to calculate the submission delay period that should be respected
+	// by the given member to avoid all members submitting the same inactivity claim
+	// at the same time.
+	inactivityClaimSubmissionDelayStepBlocks = 3
+	// inactivityClaimMaximumSubmissionBlocks determines the maximum block
+	// duration of inactivity claim submission procedure.
+	inactivityClaimMaximumSubmissionBlocks = 60
```

Comment:
What's the reasoning behind 60?

Reply:
As discussed we decided to make the synchronization context-driven. See 9ad6371c9f14856f97900d95e627f05ed29dc344.